src/cinema/domainmodel/Seats.java

```diff
@@ -0,0 +1,29 @@
+package cinema.domainmodel;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class Seats {
+
+    @JsonProperty("total_rows")
+    private final int numOfRows;
+
+    @JsonProperty("total_columns")
+    private final int numOfColumns;
+
+    @JsonProperty("available_seats")
+    private List<Seat> seats;
```

Comment:
Can you make names closer to json names?

Reply:
already done